packages/xrpl/src/client/index.ts

```diff
@@ -92,7 +92,9 @@ import {
   handleStreamPartialPayment,
 } from './partialPayment'
 
-export interface ClientOptions extends ConnectionUserOptions {
+export interface ClientOptions<
+  ClientAPIVersion extends APIVersion = typeof DEFAULT_API_VERSION,
+> extends ConnectionUserOptions {
```

Comment:
I don't think this typing change is needed

Reply:
Without this, `Client` class will not be able to recognize the API version set in the constructor, and the response of `request()` and `requestAll()` will only be the types of APIv2 (`DEFAULT_API_VERSION`).